bug(npm): Update karma-webpack when release is cut. Peer Dep issues.

You cannot currently shrinkwrap cli on master because of a karma-webpack and webpack@2.1.0-beta.x peer dependency problem. The issue is already fixed by that team, but a release needs to be cut. When @sokra does a release we need to update it in the cli.  

npm ERR! Problems were encountered
> npm ERR! Please correct and try again.
> npm ERR! peer invalid: webpack@^1.4.0, required by karma-webpack@1.7.0

--- a/package.json
+++ b/package.json
@@ -56,7 +56,7 @@
     "istanbul-instrumenter-loader": "^0.2.0",
     "json-loader": "^0.5.4",
     "karma-sourcemap-loader": "^0.3.7",
-    "karma-webpack": "^1.7.0",
+    "karma-webpack": "^1.8.0",
     "leek": "0.0.21",
     "less": "^2.7.1",
     "less-loader": "^2.2.3",

